main.py

```diff
@@ -132,7 +132,8 @@ def check_command(c, exe):
 
 
 if __name__ == '__main__':
-    thread.start_new_thread(main_tk_thread, ())
+    t = Thread(target=main_tk_thread)
```

Comment:
Did it work for you? I get
```
Traceback (most recent call last):
  File "main.py", line 163, in <module>
    _, contours, hierarchy = cv2.findContours(thresh1, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
ValueError: not enough values to unpack (expected 3, got 2)
```

Reply:
Seems like they changed API:
https://docs.opencv.org/4.1.1/d4/d73/tutorial_py_contours_begin.html
`findContours()` now returns 2 elements, not 3. But it should be fine, because the first was ignored by us either way